### ISSUE TITLE
Generated files do not end with newline

### ISSUE THREAD

**Issue by rahmant3** [fin-language/fin#52]
IAR by default will create compiler warnings for the generated files because they do not end in a newline. This is a warning that is enabled by default in IAR. 

It seems to exist because it was required for older compilers. Ref: https://stackoverflow.com/questions/72271/no-newline-at-end-of-file-compiler-warning

I think to work with older compilers, it would be a good idea to generate the files with a newline at the end.

**Comment by adamfk** [comment]
It looks like the generated files do end with a "\n". Maybe IAR is complaining because they aren't ending with a "\r\n"?

**Comment by adamfk** [comment]
With the change in https://github.com/fin-language/fin/issues/55, I think this issue may be resolved. Can you try out the default behavior in finlang 0.3.1?

**Comment by rahmant3** [comment]
This worked! My apologies, I think this one may have been partly user error. IAR was complaining about a particular generated enum file:

```c#
// SampleEnum.cs
public enum SampleEnum
{
  One,
  Two
}
```

This generated a file without a blank line as the last line. I found out I could have worked around this if I placed a blank line myself in the `.cs` file. It wasn't something I was used to having to do on the other files, which were classes.

Anyhow, with the above change, I don't have to add the blank line myself and it compiles without warning!
